tests/app/test_agenting.py

```diff
@@ -62,6 +64,52 @@ def test_load_ends(helpers):
         assert isinstance(end, agenting.QueryCollectionEnd)
 
 
+def test_load_tocks_config(helpers):
+    with helpers.openKeria() as (agency, agent, app, client):
+        agenting.loadEnds(app=app)
+        assert app._router is not None
+
+        assert agent.cfd == {
+            "dt": "2022-01-20T12:57:59.823350+00:00",
+            "keria": {
+                "dt": "2022-01-20T12:57:59.823350+00:00",
+                "curls": ["http://127.0.0.1:3902/"]
+            },
+            "EK35JRNdfVkO4JwhXaSTdV4qzB_ibk_tGJmSVcY4pZqx": {
+                "dt": "2022-01-20T12:57:59.823350+00:00",
+                "curls": ["http://127.0.0.1:3902/"]
+            },
+            "EI7AkI40M11MS7lkTCb10JC9-nDt-tXwQh44OHAFlv_9": {
+                "dt": "2022-01-20T12:57:59.823350+00:00",
+                "curls": ["http://127.0.0.1:3902/"]
+            },
+            "tocks": {
+                "initer": 0.0,
+                "escrower": 1.0
+            }
+        }
+
+        assert agent.tocks == {
+            "initer": 0.0,
+            "escrower": 1.0
+        }
+
+        escrower_doer = next((doer for doer in agent.doers if isinstance(doer, agenting.Escrower)), None)
+        assert escrower_doer is not None
+        assert escrower_doer.tock == 1.0
+
+        querier_docer = next((doer for doer in agent.doers if isinstance(doer, agenting.Initer)), None)
```

Comment:
did you intend the name 'docer' here and below? If so, what is the distinction?

Reply:
It's a typo, I meant `doer`, I'll update it. Thanks.